ddtrace/span.py

```diff
@@ -49,13 +49,13 @@ def __init__(
         span_id=None,
         parent_id=None,
         start=None,
-        ctx=None,
+        context=None,
```

Comment:
The point is that we're relying on this to keep the state of the current trace. If it's not the context, it should be something else (but I'm advocating to have just a 1-1 between Context-Trace). I think it would be good to have it as a mandatory argument because I don't think one of the use cases is to handle ``Span`` manually.

I expect users to rely in ``tracer.start_span()`` or ``tracer.trace()`` (etc...)